state/src/check.rs

```diff
@@ -363,14 +363,6 @@ fn check_deal_states_against_sectors(
             continue;
         };
 
-        acc.require(
```

Comment:
Unfortunately, I think this invariant is no longer true with these changes (you cannot deduce when a sector's deals should have started just by looking at a sector info).

Reply:
instead of removing this constraint you should keep but loosen it and assert that the deal start epoch >= the sector activation epoch.